src/index.js

```diff
@@ -1,3 +1,137 @@
+// 전역 변수
+const todoItem = `
+<div class="view">
+  <input class="toggle" type="checkbox"/>
+  <label class="label">새로운 타이틀</label>
+  <button class="destroy"></button>
+</div>
+<input class="edit" value="새로운 타이틀" />
+`;
+let state = {
+    condition: "all",
+    items: [],
+};
+const todoList = document.getElementById("todo-list");
+
+// state객체의 상태에 따라 새로고침 해주는 메소드
+// state.condition 에 따라 각각 다른 리스트를 보여준다.
+// 함수의 마지막에 localStorage에 저장하는 코드가 있음
+function rePaint(state) {
+    todoList.innerHTML = '';
+    let rePaintList = [];
+    if (state.condition === "all") rePaintList = state.items; 
+    if (state.condition === "active") rePaintList = state.items.filter(item=> item.completed === false);
+    if (state.condition === "completed") rePaintList = state.items.filter(item=> item.completed === true);
+
+    rePaintList.forEach((item,i)=>{
+        const classList = [];
+        if (item.completed) classList.push("completed");
+        if (item.editing) classList.push("editing");
+        const li = document.createElement("li");
+        li.innerHTML = todoItem;
+        li.setAttribute("class", classList.join(" "));
+        li.dataset.id = i;
+        li.querySelector(".label").textContent = item.todo;
+        li.querySelector(".edit").value = item.todo;
+        li.querySelector(".edit").addEventListener("keyup", onEdit);
+        li.querySelector(".toggle").addEventListener("change", onChangeCheckBox);
+        if (item.completed) li.querySelector(".toggle").setAttribute("checked", item.completed);
+        li.querySelector(".destroy").addEventListener("click", onClickDestroy);
+
+        li.addEventListener('dblclick',onDoubleClickTodoItem);
+        todoList.append(li);
+    });
+    document.querySelector("#new-todo-title").value = '';
+    document.querySelector(".todo-count strong").textContent = state.items.length;
+    localStorage.setItem("state", JSON.stringify(state));
+}
+
+// Events
+// 이벤트들은 전역객체 상태값을 변경하고, 함수의 마지막 부분에서 rePaint 함수를 실행시킨다.
+function onChangeCheckBox(e) {
```

Comment:
만약에 toggle이라는 행위가 change가 아닌 다른 이벤트를 통해서 발생하면 어떨까요?

```suggestion
function toggleCompleted(e) {
```
그래서 "이벤트 행위"에 초점을 맞춘 이름이 아니라 해당 이벤트가 해야할 일에 대해서 메소드 명을 작성하는것이 좋습니다.